en/04_Changelogs/5.0.0.md

```diff
@@ -13,6 +13,7 @@ title: 5.0.0 (unreleased)
   - [ORM](#api-orm)
 - [Dependency changes](#dependency-changes)
   - [General changes](#dep-general)
+  - [URLs generated by Silverstripe CMS don't include a trailing slash by default](#trailing-slash)
```

Comment:
This isn't a "Dependency change", instead it's a functional change

Reply:
Good point. I'll slap it in the "other breaking changes" section which is also being added in https://github.com/silverstripe/developer-docs/pull/137

Reply:
done